lib/tasks/webpacker.rake

```diff
@@ -23,4 +23,9 @@ namespace :webpacker do
   task :npm_install do
     system 'npm install'
   end
+
+  desc 'Invoke this task if you are sure not to install npm dependencies'
```

Comment:
i don't understand the description you mean: 
```suggestion
  desc 'Invoke this task if you are sure you don't want to install npm dependencies'
```

Reply:
Thanks for the suggestion <3 @thomasmaas 
We use that https://github.com/3scale/porta/pull/567/files#diff-1d37e48f9ceff6d8030570cd36286a61R359 because we already install the npm packages in the `dependencies_npm` task and we want to enjoy the cache.

`bundler` has `bundle check`
We could install https://www.npmjs.com/package/npm-check or https://www.npmjs.com/package/installed-check in the system-builder though instead.